Error message instead of compiler crash on incorrect use of attribute syntax.
Patch by Brian Drummond.
See also https://gna.org/support/?3060

Index: ghdl-0.30~svn20130213/ghdl/parse.adb
===================================================================
--- ghdl-0.30~svn20130213.orig/ghdl/parse.adb	2010-01-12 04:15:20.000000000 +0100
+++ ghdl-0.30~svn20130213/ghdl/parse.adb	2013-12-10 22:26:56.874533000 +0100
@@ -3984,8 +3984,12 @@
          when Iir_Kind_Simple_Name
            | Iir_Kind_Selected_Name =>
             Set_Implementation (Call, Name);
+         when Iir_Kind_Attribute_Name =>        -- Support issue 3060
+            Error_Msg_Parse ("Attribute cannot be applied to procedure call");
          when others =>
-            raise Internal_Error;
+            -- Support issue 2686 : no testcase, at least improve the error message
+            Error_Kind("parenthesis_name_to_procedure_call", Name);
+            -- raise Internal_Error;
       end case;
       return Res;
    end Parenthesis_Name_To_Procedure_Call;
